app/state/reducers/ui/searchReducer.js

```diff
@@ -89,6 +92,40 @@ function searchReducer(state = initialState, action) {
       return state.merge({ unitId: null });
     }
 
+    case 'FILTER_BY_NAME': {
+      const { lang, resources } = action.payload;
+      const sortedResources = orderBy(resources, resource => resource.name[lang], ['asc']);
```

Comment:
This is a huge no go. The sorting should be done by api, not the frontend.

Reply:
If the `resources` endpoint doesn't support sorting then it needs to be requested from respa team.